### PR TITLE
CFE-1099: Add release configuration for ansible-operator-plugins

### PR DESCRIPTION
**Pull request by chiragkyal**
Add release configuration for [ansible-operator-plugins](https://github.com/openshift/ansible-operator-plugins)

- Depends on https://github.com/openshift/ansible-operator-plugins/pull/5

### CODE REVIEW

File: core-services/prow/02_config/openshift/ansible-operator-plugins/_pluginconfig.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+approve:
+- commandHelpLink: ""
+  repos:
+  - openshift/ansible-operator-plugins
+  require_self_approval: false
+lgtm:
```
Comment (arkadeepsen) [inline core-services/prow/02_config/openshift/ansible-operator-plugins/_pluginconfig.yaml:6]: Can you add the `label` stanza similar to the following?
https://github.com/openshift/release/blob/ad138117d24f1ecade2f6fac1b43f24da0a0f81b/core-services/prow/02_config/openshift/ocp-release-operator-sdk/_pluginconfig.yaml#L6-L22

Add Trilok to the list of `allowed_users` for `backport-risk-assessed` label.

File: ci-operator/config/openshift/ansible-operator-plugins/OWNERS [hunk 2]
```diff
@@ -0,0 +1,10 @@
+approvers:
+  - joelanford
+  - everettraven
+  - oceanc80
+  - TrilokGeer
+reviewers:
+  - joelanford
+  - everettraven
+  - oceanc80
+  - TrilokGeer
```
Comment (arkadeepsen) [inline ci-operator/config/openshift/ansible-operator-plugins/OWNERS:10]: You can add both your and my github usernames in the reviewers list.

File: ci-operator/jobs/openshift/ansible-operator-plugins/OWNERS [hunk 3]
```diff
@@ -0,0 +1,10 @@
+approvers:
+  - joelanford
+  - everettraven
+  - oceanc80
+  - TrilokGeer
+reviewers:
+  - joelanford
+  - everettraven
+  - oceanc80
+  - TrilokGeer
```
Comment (arkadeepsen) [inline ci-operator/jobs/openshift/ansible-operator-plugins/OWNERS:10]: Same as https://github.com/openshift/release/pull/54964#discussion_r1698762065.

File: core-services/prow/02_config/openshift/ansible-operator-plugins/OWNERS [hunk 4]
```diff
@@ -0,0 +1,10 @@
+approvers:
+  - joelanford
+  - everettraven
+  - oceanc80
+  - TrilokGeer
+reviewers:
+  - joelanford
+  - everettraven
+  - oceanc80
+  - TrilokGeer
```
Comment (arkadeepsen) [inline core-services/prow/02_config/openshift/ansible-operator-plugins/OWNERS:10]: Same as https://github.com/openshift/release/pull/54964#discussion_r1698762065.

**Comment by arkadeepsen** [comment]
Removing the label till the above changes are included
/lgtm cancel

File: ci-operator/config/openshift/ansible-operator-plugins/OWNERS (hunk 2 above)
Comment (chiragkyal) [reply ci-operator/config/openshift/ansible-operator-plugins/OWNERS:10]: Done, added

File: core-services/prow/02_config/openshift/ansible-operator-plugins/OWNERS (hunk 4 above)
Comment (chiragkyal) [reply core-services/prow/02_config/openshift/ansible-operator-plugins/OWNERS:10]: Done, added

**Comment by chiragkyal** [comment]
`ci/rehearse/openshift/ansible-operator-plugins/main/images` is green now

**Pull request merged**